packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts

```diff
@@ -322,6 +322,7 @@ export default ({
           })
         )
         yield put(actions.form.stopSubmit('previewSell'))
+        yield put(actions.components.refresh.refreshClicked())
```

Comment:
might want to wrap in a `delay` for a couple seconds just to ensure backend is updated and returns proper data